torch/csrc/torch_api.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
We can’t do this unless we rename it to torch.h in setup.py. The header must not change for users, otherwise all C++ extensions will break 2 weeks after releasing them

Reply:
This change was just to separate the python/non-python parts of this header so that it can be included in no-python files. I will rename torch_api.h to something more descriptive.